integrations/waffle-mars.md

```diff
@@ -0,0 +1,402 @@
+
+9. Next, you can go back and edit the test to check for 10 tokens:
+```typescript
+it('Mints the correct initial balance', async () => {
+  expect(await token.balanceOf(wallet.address)).to.equal(10); // This should pass
+});
+```
+10. If you run the tests again, you should now see one passing test:
+```
+npm run test
+```
+
+11. You've tested the ability to mint tokens, next you'll test the ability to transfer the minted tokens. If you want to write a failing test first again that is up to, however the final test should look like this:
+```typescript
+it('Should transfer the correct amount of tokens to the destination account', async () => {
+  // Send the destination wallet 7 tokens
+  await (await token.transfer(walletTo.address, 7)).wait();
+
+  // Expect the destination wallet to have received the 7 tokens
+  expect(await token.balanceOf(walletTo.address)).to.equal(7);
+});
+```
+
+Congratulations, you should now have two passing tests! Altogether, your test file should look like this:
+```typescript
+import { use, expect } from 'chai';
+import { Provider } from '@ethersproject/providers';
+import { solidity } from 'ethereum-waffle';
+import { ethers, Wallet } from 'ethers';
+import { MyToken, MyTokenFactory } from '../build/types';
+
+use(solidity);
+
+describe ('MyToken', () => {
+  let provider: Provider = new ethers.providers.JsonRpcProvider('https://rpc.testnet.moonbeam.network');
+  let wallet: Wallet;
+  let walletTo: Wallet;
+  let token: MyToken;
+
+  beforeEach(async () => {
+    const PRIVATE_KEY = '<insert-your-private-key-here>'
+    wallet = new Wallet(PRIVATE_KEY).connect(provider);
+    walletTo = Wallet.createRandom().connect(provider);
+    token = await new MyTokenFactory(wallet).deploy();
+    let contractTransaction = await token.initialize(10);
+    await contractTransaction.wait();
+  });
+
+  it('Mints the correct initial balance', async () => {
+    expect(await token.balanceOf(wallet.address)).to.equal(10);
+  });
+
+  it('Should transfer the correct amount of tokens to the destination account', async () => {
+    await (await token.transfer(walletTo.address, 7)).wait();
+    expect(await token.balanceOf(walletTo.address)).to.equal(7);
+  });
+})
+```
+
+If you want to write more tests on your own, you could consider testing transfers from accounts without any funds or transfers from accounts without enough funds.
+
+You've used Waffle to write, compile, and test the `MyToken` contract and can now move on to deploying the contract to Moonbase Alpha with Mars. 
+
+## Use Mars to Deploy to Moonbase Alpha
+
+After you compile your contracts and before deployment, you will have to generate contract artifacts for Mars. Mars uses the contract artifacts for typechecks in deployments. Then you'll need to create a deployment script and deploy the `MyToken` smart contract.
+
+Remember, you will be deploying to Moonbase Alpha and will need to use the testnet configurations:
```

Comment:
All "testnet" instances should be "TestNet" (nitpicking but to keep consistency throughout the website)